__tests__/app/api/user-metrics/route.test.ts

```
@@ -0,0 +1,355 @@
+  }
+  async json() {
+    if (this.body == null) return this.body as null;
+    if (typeof this.body === 'string') {
+      try {
+        return JSON.parse(this.body);
+      } catch {
+        return this.body;
+      }
+    }
+    // Good enough for tests; real Response.json() returns any.
+    return this.body as unknown as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)